biome.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.0/schema.json",
+  "formatter": {
+    "enabled": false
+  },
+  "organizeImports": {
+    "enabled": true,
+    "ignore": [
```

Comment:
Can we reuse the .gitignore file, so we do not need to configure so many ignore rules

https://github.com/web-infra-dev/rsbuild/blob/main/biome.json#L14-L19